examples/e2e/qwen3_8b_sharegpt_ultrachat.py

```diff
@@ -0,0 +1,54 @@
+import sys
+from pathlib import Path
+
+# Add scripts directory to path so we can import the run_e2e function.
```

Comment:
This is great and exactly what we'd want for well-lit paths.

We should add a script for llama3

Reply:
Added an example script for llama3. I decided to make this one just use 5k samples from Sharegpt. It's not enough for a full training run but enough to see the drafter learning something. I also added some comments to this script + the qwen one describing their uses.